documentation/modules/oauth/con-oauth-authorization-keycloak-example.adoc

```diff
@@ -0,0 +1,452 @@
+// Module included in the following module:
+//
+// con-oauth-authorization-keycloak-authorization-services.adoc
+
+[id='con-oauth-authorization-keycloak-example_{context}']
+= Keycloak Authorization Services Example
```

Comment:
Can we introduce what we're trying to achieve with this example?

Reply:
Hmmm ... It's an end to end example that demonstrates how to configure permissions. It's actually the more important addition in terms of helping users to use Keycloak Authorization Services for authorization. It is a 'port' of examples/README-authz.md from Docker to Strimzi and was requested by users. As such it is written in a different style as the regular documentations.

I don't know how to address the sections part but the introduction could be something like:
```suggestion
= Keycloak Authorization Services Example

This is an end-to-end example of using Keycloak Authorization Services to configure authorization rules for use with `keycloak` authorization in Strimzi.
The example starts by deploying the Keycloak server with pre-configured realm, requiring no additional configuration.
We deploy the Kafka cluster configured to use one of the pre-configured realms.
We connect to the Keycloak Admin Console and demonstrate the sections of the UI where authorizations rules are configured.
We use Kafka CLI client tools with different personal and service accounts to demonstrate limited access based on the permissions granted to different accounts.

```

Reply:
Great. Looks good Marko. Couple of things: _with pre-configured realm**s**_? And we use _user interface_ rather than _UI_. :+1: